ex9.gen.json

```diff
@@ -0,0 +1,6 @@
+[
+  {"bn":"urn:dev:ow:10e2073a01080063:"},
```

Comment:
Is the colon in the end of ow urn intentional? Other examples don't have it (except ex9)